src/DBUtils/sqlite_db_utils.jl

```diff
@@ -69,19 +69,19 @@ function insert_row!{T}(db::SQLite.DB, tablename, data_values::Dict{Symbol, T},
 end
 
 """
-disable_foreing_checks(con::SQLite.DB)
-Disables foreing checks for SQLite database
+disable_foreign_checks(con::SQLite.DB)
+Disables foreign checks for SQLite database
 """
-function disable_foreing_checks(conn::SQLite.DB)
+function disable_foreign_checks(conn::SQLite.DB)
     SQLite.execute!(conn, "PRAGMA foreign_keys = OFF")
 
 end
 
 """
-    enable_foreing_checks(con::SQLite.DB)
-Enables foreing checks for SQLite database
+    enable_foreign_checks(con::SQLite.DB)
+Enables foreign checks for SQLite database
 """
-function enable_foreing_checks(conn::SQLite.DB)
+function enable_foreign_checks(conn::SQLite.DB)
     SQLite.execute!(conn, "PRAGMA foreign_keys = ON")
 
-end
\ No newline at end of file
+end
```

Comment:
equivalent of set_innodb_checks for sqlite?

Reply:
set_innodb_checks is really only for use with load_medline as of now due to the volume of the loads.  And load_medline only supports MySQL (due to the volume of the loads).  Should I find/add an equivalent function for SQLite so there is symmetry even though it likely won't be used?